ios/FluentUI/Avatar/Persona.swift

```diff
@@ -42,6 +42,10 @@ public protocol Persona {
 
     /// The subtitle value for the persona.
     var subtitle: String { get }
+
+    /// Loads the image asynchronously. Implement this instead of use ``image``  property if it needs to be fetched asynchronously. Image caching should be handled in the implementation for optimial efficiency.
+    /// - Parameter completion: The completion block that returns the image
+    @objc optional func loadImageAsync(completion: @escaping (UIImage?) -> Void)
```

Comment:
I think we should be explicit about how this interacts with the image property. What happens if you provide both? Are there potential race conditions? Should prevent both being implemented simultaneously (via `preconditionFailure`)?

I would lean towards allowing both as showing a static placeholder image that is replaced by an async loaded image seems like a normal workflow, but I think we should be explicit about it. The use of `instead of` in the comment above makes me think they are mutually exclusive, but the implementation looks like it would allow providing a placeholder image via `image` and the "real" image via `loadImageAsync`

Reply:
Updated comment. Grammar is definitely one of my weak areas. So please proofreading for me again and feel free to reword the sentence as you see fit. 😊